client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.alibaba.nacos.client.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TenantUtilTest {
+    
+    @Test
+    public void testGetUserTenantForAcm() {
+        String expect = "test";
+        System.setProperty("acm.namespace", expect);
+        String actual = TenantUtil.getUserTenantForAcm();
+        Assert.assertEquals(expect, actual);
+        System.clearProperty("acm.namespace");
```

Comment:
If assert fail, the reset will invalid.

you can do this in After method.

Reply:
ok, i will fix this

Reply:
This case do not fix to clear env in after.